examples/hybrid/tuning/mwe_tune_ke.jl

```diff
@@ -0,0 +1,249 @@
+using CUDA
+using ClimaComms
+using ClimaCore
+using LinearAlgebra
```

Comment:
Where is this import being used? Do you know if this makes CUDA use cuBLAS for linalg operations on cuArrays?

Reply:
I believe all our current operations are thread-local! I do not believe we are currently using cuBLAS